astropy/utils/masked/core.py

```diff
@@ -1078,10 +1080,20 @@ def argsort(self, axis=-1, kind=None, order=None):
 
         return np.lexsort(keys, axis=axis)
 
-    def sort(self, axis=-1, kind=None, order=None):
-        """Sort an array in-place. Refer to `numpy.sort` for full documentation."""
+    def sort(self, axis=-1, kind=None, order=None, *, stable=False):
+        """Sort an array in-place. Refer to `numpy.sort` for full documentation.
+
+        Notes
+        -----
+        Masked items will be sorted to the end. The implementation
+        is via `np.lexsort` and thus ignores the ``kind`` and ``stable`` arguments;
```

Comment:
Darn, this should have been `numpy.lexsort` for readthedocs to be happy...

Reply:
Since it must be getting late in Europe, I took the liberty to fix this directly on this branch. Hope you all don't mind!

Reply:
Thanks @pllim !